### ISSUE TITLE
[accessibility] The × character in the close button of popups should be explicitly hidden from ATs

### ISSUE THREAD

**Issue by Malvoz** [maplibre/maplibre-gl-js#360]
Although the contents of `aria-label` overrides the × character in the close button of popups, it is still announced as <q>multiplication</q> or <q>times</q> (depending on the screen reader) when the user is hovering the control with a mouse pointer.

To solve this issue, the × character:

https://github.com/maplibre/maplibre-gl-js/blob/9e6d57aaf7f77161c3d2f9dfd330a16227b60f19/src/ui/popup.ts#L497

should be wrapped in `<span aria-hidden="true">`.

(This would also align and be more consistent with other controls that hide icons from <span title="Assistive technology">ATs</span>, such as the zoom in control)

https://github.com/maplibre/maplibre-gl-js/blob/e78ad7944ef768e67416daa4af86b0464bd0f617/src/ui/control/navigation_control.ts#L58

<hr>

(The same issue was also reported with MapBox: https://github.com/mapbox/mapbox-gl-js/issues/11035.)
